ui/src/song/SongList.js

```diff
@@ -79,14 +79,14 @@ const SongList = (props) => {
       >
         {isXsmall ? (
           <SimpleList
-            primaryText={(r) => r.title}
-            secondaryText={(r) => r.artist}
-            tertiaryText={(r) => (
+            primaryText={(r) => r.title.slice(0, 35)}
+            secondaryText={(r) => (
               <>
+                <span>{r.artist} &#8226; </span>
```

Comment:
Can you align this one on the right?

Reply:
I assume you are saying to shift the author name to right side
@jvoisin

Reply:
Err, I meant shifting the duration to the right side, sorry.

Reply:
It would be nice if the interactive options  are at the right side like star and non interactive on the left [author and timestamp] we could give more whitespace between author and timestamp to make it more user friendly